src/Makevars

```diff
@@ -41,10 +41,10 @@ ifeq ($(USE_TBB),  Windows)
   # compiler: overwrite default (which is cl = MS compiler)
   MAKE_ARGS += rtools=true compiler=gcc
   ifeq ("$(WIN)", "64")
-    # TBB defaults to ia32
     MAKE_ARGS += arch=intel64
     ARCH_DIR=x64/
   else
+    MAKE_ARGS += arch=ia32
```

Comment:
I needed to add this to avoid `undefined reference` errors during link, for some reason.